web/src/user/UserInterface.ts

```diff
@@ -192,85 +193,18 @@ export class UserInterface extends Interface {
                         <a href="#/" class="pf-c-page__header-brand-link">
                             <img
                                 class="pf-c-brand"
-                                src="${first(this.brand?.brandingLogo, DefaultBrand.brandingLogo)}"
-                                alt="${(this.brand?.brandingTitle, DefaultBrand.brandingTitle)}"
+                                src="${this.brand.brandingLogo}"
+                                alt="${this.brand.brandingTitle}"
                             />
                         </a>
                     </div>
                     <div class="pf-c-page__header-tools">
                         <div class="pf-c-page__header-tools-group">
-                            ${this.uiConfig.enabledFeatures.apiDrawer
-                                ? html`<div
-                                      class="pf-c-page__header-tools-item pf-m-hidden pf-m-visible-on-lg"
-                                  >
-                                      <button
-                                          class="pf-c-button pf-m-plain"
-                                          type="button"
-                                          @click=${() => {
```

Comment:
This duplicates the functionality of the `window.EVENT_API_DRAWER_TOGGLE` functionality above, so replacing this with "send the `EVENTS_API_DRAWER_TOGGLE` event to `window`" eliminates that duplication.